crates/wasm-rquickjs/src/typescript.rs

```diff
@@ -720,7 +719,7 @@ impl DtsWriter {
     }
 
     pub fn begin_export_async_function<'a>(&'a mut self, name: &str) -> DtsFunctionWriter<'a> {
-        self.indented_write(format!("export async function {name}("));
+        self.indented_write(format!("export function {name}("));
```

Comment:
Small unrelated fix. Async is purely an implementation concern (which allows awaiting in the body of the function) and is not allowed in type definition files.

Making the return type a promise is enough to make the function "async"